tensorflow/core/grappler/optimizers/generic_layout_optimizer_transposer_factory.cc

```diff
@@ -27,6 +27,9 @@ std::shared_ptr<Transposer> TransposerFactory::GetTransposer(
     return GetOrCreateIfNotFound<DefaultLayoutSensitiveOpTransposer>(
         "DefaultLayoutSensitiveOp");
   }
+  if (IsBiasAdd(node)) {
```

Comment:
This should be restricted to `BiasAdd` also and not include `BiasAddV1`.

And can this be moved to below AvgPoolGrad?

Reply:
Done. PTAL.